spec/classes/vault_spec.rb

```diff
@@ -390,23 +390,70 @@
       :architecture              => 'x86_64',
       :kernel                    => 'Linux',
     }}
-    context 'includes systemd init script' do
+    context 'includes systemd init script with systemd_version < 229' do
+      let(:facts) {{
+                   :systemd_version => 228
```

Comment:
Some funky indentation in all the `:facts` blocks you've added, can you de-dent to match.

Reply:
Will do.